.github/workflows/ce-dev-publish-docs.yml

```diff
@@ -41,10 +41,10 @@ jobs:
           git clone https://github.com/codeenigma/wikis2pages.git
           cd wikis2pages
           /bin/bash init.sh https://${{ secrets.GITHUB_TOKEN }}@github.com/codeenigma/ce-dev.git 1.x
-          /bin/sh set-current.sh ce-dev-1.x
+          /bin/sh set-current.sh ce-dev-local
```

Comment:
How come this change? :slightly_smiling_face:

Reply:
devel branch seems outdated if you compare with 1.x. The feature branch was created from 1.x.
https://github.com/codeenigma/ce-dev/blob/1.x/.github/workflows/ce-dev-publish-docs.yml#L44